src/plugins/vis_type_timeseries/public/application/components/vis_editor_visualization.js

```diff
@@ -98,26 +102,16 @@ class VisEditorVisualizationUI extends Component {
     }
   };
 
-  hasShowPanelIntervalValue() {
+  isAnyPanelTypeExceptTimeseries() {
     const type = get(this.props, 'model.type', '');
-    const interval = get(this.props, 'model.interval', AUTO_INTERVAL);
-
-    return (
-      [
-        PANEL_TYPES.METRIC,
-        PANEL_TYPES.TOP_N,
-        PANEL_TYPES.GAUGE,
-        PANEL_TYPES.MARKDOWN,
-        PANEL_TYPES.TABLE,
-      ].includes(type) &&
-      (isAutoInterval(interval) || isGteInterval(interval))
-    );
-  }
 
-  getFormattedPanelInterval() {
-    const interval = convertIntervalIntoUnit(this.state.panelInterval, false);
-
-    return interval ? `${interval.unitValue}${interval.unitString}` : null;
+    return [
```

Comment:
nit: I see you renamed that method to `isAnyPanelTypeExceptTimeseries` maybe in that case instead of 

```
 [	
        PANEL_TYPES.METRIC,	
        PANEL_TYPES.TOP_N,	
        PANEL_TYPES.GAUGE,	
        PANEL_TYPES.MARKDOWN,	
        PANEL_TYPES.TABLE,	
      ].includes(type) &&
```
we can writee just `type !== PANEL_TYPES.TIMESERIES `